mxnet-predict.spec

```diff
@@ -17,11 +21,10 @@ rm -rf ../build; mkdir ../build; cd ../build
 # use LAPACK functions in OpenBLAS:
 # manually set MXNET_USE_LAPACK=1 and turn off USE_LAPACK in cmake
 export CFLAGS="-DMXNET_USE_LAPACK=1 -DMXNET_THREAD_LOCAL_ENGINE=1"
+export PYTHONV=$(echo $PYTHON_VERSION | cut -f1,2 -d.)
+export PYTHON3V=$(echo $PYTHON3_VERSION | cut -f1,2 -d.)
 
 cmake ../%{n}-%{realversion} -GNinja \
-    -DCMAKE_CUDA_COMPILER_LAUNCHER=ccache \
-    -DCMAKE_C_COMPILER_LAUNCHER=ccache \
-    -DCMAKE_CXX_COMPILER_LAUNCHER=ccache \
```

Comment:
pass `-DCMAKE_INSTALL_LIBDIR=lib` to get lib instead of lib64. fix `-toolfile` to pointto lib